### PR TITLE
fix(@desktop/chat): Show badge and message in activity center for reply elements

### LINKED ISSUE #5523: Replies list in the Activity centre does not display community badge / icon
# Bug Report

## Description

Currently there are no Community badges / icons displayed against replies in the Activity Centre

The user must be able to identify which Community & channel the notified reply is at.

#### Expected behavior
The Community badge & channel name must be shown in the Replies list

#### Actual behavior
image

### Additional Information

- Status desktop version: Master : 9c9d6330650312378901becb13584b1a6d4926a1

### PR DESCRIPTION
**Pull request by endulab**
Fix #5523

### What does the PR do

Shows badge with message text for reply elements.

### Affected areas

Chat / Activity center

### Screenshot of functionality

image

### CODE REVIEW

**Comment by endulab** [comment]
@jrainville can you test ?

File: src/app/modules/main/activity_center/controller.nim [hunk 1]
```diff
@@ -116,4 +116,11 @@ proc getChannelGroups*(self: Controller): seq[ChannelGroupDto] =
...
-  return self.chatService.getOneToOneChatNameAndImage(chatId)
\ No newline at end of file
+  return self.chatService.getOneToOneChatNameAndImage(chatId)
+
+proc getMessageById*(self: Controller, chatId, messageId: string): MessageDto =
+  let (message, _, err) = self.messageService.getDetailsForMessage(chatId, messageId)
+  if(err.len == 0):
+    return message
+  else:
+    return MessageDto()
```
Comment (jrainville) [inline src/app/modules/main/activity_center/controller.nim:126]: It's a bit cleaner
```suggestion
  if err.len > 0:
    return MessageDto()
    
  return message
    
```

**Review by jrainville — COMMENTED** [review]
Code looks good. Testing now

**Review by jrainville — APPROVED** [review]
Good job, the feature works very well!

image

The only issues I found were minor or not related to your PR
1. On long text, the bubble could use some padding on the right. See attached image. Can be fixed in another PR or right now if it's simple enough

image

2. The notification bell at the top doesn't show the number of unread notifications, but that's also found on master.  I'll open an issue for that

File: src/app/modules/main/activity_center/controller.nim (hunk 1 above)
Comment (endulab) [reply src/app/modules/main/activity_center/controller.nim]: Good catch. Fixed.

**Comment by endulab** [comment]
@jrainville re 1. It is not so trivial. I was fighting with this. Unfortunately this is StyledTextEdit (TextEdit). There is no elide (abc...) option. Also it keeps rich html text. I tried to use TextMetrics but w/o success. I am not sure why StyledText is not used. Do we ever edit badge in activity center ? I guess not. We should have a separated issue for this.

**Pull request merged**